### ISSUE TITLE
Validation set accuracy every k epochs

### ISSUE THREAD

**Issue by AndreiBarsan** [AndreiBarsan/visualqa#5]
We can either do this, or split the training set into something 95%-5%, and only used validation for occasional validation. 

What do you guys think?
